docs/api-reference/widgets/geolocate-widget.md

```diff
@@ -0,0 +1,67 @@
+import {WidgetPreview} from '@site/src/doc-demos/widgets';
+import {_GeolocateWidget} from '@deck.gl/widgets';
+
+# GeolocateWidget (Experimental)
+
+<img src="https://img.shields.io/badge/from-v9.2-green.svg?style=flat-square" alt="from v9.2" />
+
+This widget allows users to input geographic coordinates (decimal or DMS format) and fly the view to that location. Enter coordinates in the text box and click "Go" or press Enter.
```

Comment:
Geolocate typically refers to detecting the user’s current location with an API - not manually typing in coordinates.

I'd prefer we renamed this to something like CoordinateWidget, LocationWidget, ViewStateWidget, JumpToWidget, FlyToWidget, etc.. and reserve GeolocateWidget for this other functionality. 

Any of these alternatives is a much better match for this description. My top pick is either `CoordinateWidget` or `JumpToWidget`